tools/travis/docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openwhisk/nodejs6action
```

Comment:
Do we only expect nodejs?

Reply:
yes, we're building on top of this dockerfile to run the package action.